frontend/src/components/Firebase.js

```diff
@@ -188,19 +188,55 @@ class Firebase {
   }
 
   // Request event edit given the parameters. 
- // all parameters are optional except credentials and eventId
- requestEventUpdate(eventId, title, date, startTime, endTime, description, plusCode, location, locationName, files = "", category, organization = "", invitedAttendees = "", uid) {
-    
-    //To be completed by John
-    console.log("update")
+  // all parameters are optional except uid and eventId
+  requestEventUpdate(eventId, uid, eventName = null, date = null, startTime = null, endTime = null, description = null, location = null, locationName = null, imagePaths = null, category = null, organization = null) {
+    var requestId = this.generateRequestId();
+    var path = this.sessionId + "/" + requestId;
+
+    // Send in a request
+    this.sessionsRef.ref('REQUESTS').child(path).set({
+      code: 6,
```

Comment:
sure would be nice if we knew what this code meant without having to poke around!! :P